### ISSUE TITLE
`PythonFFTImageFilterTest` failing in Windows CI

### ISSUE THREAD

**Issue by tbirdso** [InsightSoftwareConsortium/ITK#3057]
### Description

Windows CI is failing after #3002 due to issue in FFT test

[CDash results](https://open.cdash.org/test/582543193)

### Steps to Reproduce

[Windows CI on Azure](https://dev.azure.com/itkrobotwindowpython/ITK.Windows.Python/_build/results?buildId=2786&view=results)

### Expected behavior

`ComplexToRealImageFilter` is available and all CI passes

### Actual behavior

```
Running itkImageFileWriterICF2... done
Traceback (most recent call last):
  File "D:/a/1/s/Modules/Filtering/FFT/wrapping/test/FFTImageFilterTest.py", line 45, in <module>
    realFilter = itk.ComplexToRealImageFilter[ComplexImageType, ImageType].New(fftFilter)
  File "D:\a\1\s-build\Wrapping\Generators\Python\itk\support\lazy.py", line 130, in __getattribute__
    value = types.ModuleType.__getattribute__(self, attr)
AttributeError: module 'itk' has no attribute 'ComplexToRealImageFilter'
itkTestDriver: Process exited with return value: 1
```

### Reproducibility

Appears in `master` branch throughout several merges, but did not appear in CI checks for #3002 prior to merge and does not seem to appear locally

### Versions

`main` branch

### Environment

Windows CI

### Additional Information

#3002 which seems to have introduced the issue dealt with C++ TestDriver changes but not wrappings, so I'm a little confused as to how this error appeared from there. Will investigate.

**Comment by tbirdso** [comment]
Haven't been able to recreate on my Windows setup after rebuilding against `master`. ` PythonFFTImageFilterTest` passes, I can instantiate `itk.ComplexToRealImageFilter` just fine, and [`itkComplexToRealImageFilter.wrap`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/ImageIntensity/wrapping/itkComplexToRealImageFilter.wrap) has not been touched in years.

It would be helpful to have another set of eyes on this, maybe I'm missing something obvious. @dzenanz if you have a moment could you weigh in on this issue?

**Comment by dzenanz** [comment]
Let's wait for nightly builds and see whether my computer also chokes on it. If so, I could try to fix it.

Oops, we don't have a nightly Windows Python build. Rebasing would probably require a massive rebuild. I could do that overnight.

**Comment by tbirdso** [comment]
Thank you! I'll give it another pass as well and see if I can spot anything.

**Comment by tbirdso** [comment]
@dzenanz Checking back in, did you have a chance to try rebuilding the `main` branch on your local Windows machine? I'm still having difficulty recreating this issue.

**Comment by dzenanz** [comment]
I cannot reproduce it:
```text
C:\Dev\ITK-py>C:\Dev\ITK-py\Wrapping\Generators\Python\itk\itkTestDriver.exe "--add-before-env" "PYTHONPATH" "C:/Dev/ITK-py/Wrapping/Generators/Python" "--add-before-env" "PYTHONPATH" "C:/Dev/ITK-py/Wrapping/Generators/Python/itk" "--add-before-libpath" "C:/Dev/ITK-py/Wrapping/Generators/Python/itk" "--compare" "C:/Dev/ITK-py/Testing/Temporary/PythonFFTImageFilterRealTest.png" "C:/Dev/ITK-py/ExternalData/Modules/Filtering/FFT/wrapping/test/Baseline/PythonFFTImageFilterRealTest.png" "--compare" "C:/Dev/ITK-py/Testing/Temporary/PythonFFTImageFilterImaginaryTest.png" "C:/Dev/ITK-py/ExternalData/Modules/Filtering/FFT/wrapping/test/Baseline/PythonFFTImageFilterImaginaryTest.png" "C:/Program Files/Python37/python.exe" "C:/Dev/ITK-git/Modules/Filtering/FFT/wrapping/test/FFTImageFilterTest.py" "C:\Dev\ITK-vs19\ExternalData\Modules\Filtering\LabelMap\test\Input\cthead1.png" "C:/Dev/ITK-py/Testing/Temporary/PythonFFTImageFilterRealTest.png" "C:/Dev/ITK-py/Testing/Temporary/PythonFFTImageFilterImaginaryTest.png"
Loading ITKPyBase... done
Loading ITKCommon... done
Loading ITKIOImageBase... done
Loading ITKPyUtils... done
Loading ITKStatistics... done
Loading ITKImageFilterBase... done
Loading ITKTransform... done
Loading ITKImageFunction... done
Loading ITKImageGrid... done
Loading ITKFFT... done
Running itkImageFileReaderIF2... done
Running itkForwardFFTImageFilterIF2ICF2... done
Running itkForwardFFTImageFilterIF2ICF2... done
Running itkImageFileWriterICF2... done
Loading ITKPath... done
Loading ITKMesh... done
Loading ITKSpatialObjects... done
Loading ITKImageCompose... done
Loading ITKImageStatistics... done
Loading ITKImageIntensity... done
Running itkForwardFFTImageFilterIF2ICF2... done
Running itkComplexToRealImageFilterICF2IF2... done
done
done
done
done
done
done
done
done
done
done
done
done
Running itkRescaleIntensityImageFilterIF2IUC2... done
done
done
done
done
done
done
done
done
done
Running itkImageFileWriterIUC2... done
Running itkForwardFFTImageFilterIF2ICF2... done
Running itkComplexToImaginaryImageFilterICF2IF2... done
done
done
done
done
done
done
done
Running itkRescaleIntensityImageFilterIF2IUC2... done
done
done
done
done
done
done
done
done
Running itkImageFileWriterIUC2... done
Exception detected while reading C:/Dev/ITK-py/Testing/Temporary/PythonFFTImageFilterRealTest.png :  Could not create IO object for reading file C:/Dev/ITK-py/ExternalData/Modules/Filtering/FFT/wrapping/test/Baseline/PythonFFTImageFilterRealTest.png
The file doesn't exist.
Filename = C:/Dev/ITK-py/ExternalData/Modules/Filtering/FFT/wrapping/test/Baseline/PythonFFTImageFilterRealTest.png
<DartMeasurement name="BaselineImageName" type="text/string">INVALID_BASELINE_GIVEN</DartMeasurement>
Exception detected while reading C:/Dev/ITK-py/Testing/Temporary/PythonFFTImageFilterImaginaryTest.png :  Could not create IO object for reading file C:/Dev/ITK-py/ExternalData/Modules/Filtering/FFT/wrapping/test/Baseline/PythonFFTImageFilterImaginaryTest.png
The file doesn't exist.
Filename = C:/Dev/ITK-py/ExternalData/Modules/Filtering/FFT/wrapping/test/Baseline/PythonFFTImageFilterImaginaryTest.png
<DartMeasurement name="BaselineImageName" type="text/string">INVALID_BASELINE_GIVEN</DartMeasurement>

C:\Dev\ITK-py>
```
I did not build tests, so the baseline image is not downloaded. Hence the failure at the comparison stage.

**Comment by tbirdso** [comment]
Some notes from additional investigation:
- #3002 adds ITKFFT as a dependency for ITKTestKernel
- ITKTestKernel is enabled and built as a dependency for ITKCommon
- [Windows Python testing](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml) restricts the modules that are built to the ITKCommon group + ITKBridgeNumPy + all dependencies therein. This was done to reduce the amount of time a single CI run takes, as we are limited to 6 hours on an Azure instance.
- `itk.ComplexToRealImageFilter` is part of the ITKImageIntensity module (_not_ ITKFFT)

From these facts we find the following:
- ITKFFT and ITKImageIntensity (including `itk.ComplexToRealImageFilter`) were not previously built in Windows CI (see [log](https://dev.azure.com/itkrobotwindowpython/ITK.Windows.Python/_build/results?buildId=2784&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3))
- As of #3002 ITKFFT is built but ITKImageIntensity is not built in Windows CI (see [log](https://dev.azure.com/itkrobotwindowpython/a2cc6bea-a502-416d-adf3-3aa4d21ad7ff/_apis/build/builds/2786/logs/16))

The following changes are proposed and will be addressed in #3074 : 
- Explicitly turn on ITKImageIntensity in Windows Python CI for completeness
- Add ITKImageIntensity as a `TEST_DEPENDS` dependency for ITKFFT

Thank you @dzenanz for helping point me in the right direction on this investigation!
